app/helpers/user_mailer_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/MethodLength
+
+module UserMailerHelper
+  def text_key_for_step(old_step, current_step)
```

Comment:
These are `statuses`, not `steps`.
I would rename this for `status_change_text(old_status, new_status)` or smthg like that.

Reply:
Yes true that